src/style.css

```
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.search-one,
+.search-two {
```

```suggestion
.input {
```
Можна використати мікс класів (`class="input input-big"` та `class="input input-small"`)
Для input прописати загальні стилі для обох, а для великого і маленького прописати окремо